packages/atomic/cypress/integration/result-list/result-list-utils.ts

```diff
@@ -5,36 +5,46 @@ import {
 } from '../../../src/components/atomic-result/atomic-result-display-options';
 import {resultListComponent} from './result-list-selectors';
 
+interface WithAnySectionnableResultListOptions {
+  viewports?: Record<string, number>;
+  layouts?: ResultDisplayLayout[];
+  imageSizes?: ResultDisplayImageSize[];
+  densities?: ResultDisplayDensity[];
+}
+
 export function withAnySectionnableResultList(
   assertions: (
     display: ResultDisplayLayout,
     imageSize: ResultDisplayImageSize,
     density: ResultDisplayDensity
-  ) => void
+  ) => void,
+  options?: WithAnySectionnableResultListOptions
```

Comment:
I wanted to limit `it` blocks to only relevant ones, especially given that each `it` blocks count towards Cypress' limit.